src/components/SpecialResolution/InstructionalText.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <article class="instructional-text section-container">
+    Before submitting this filing, you must pass a
+    <v-tooltip
+      top
+      content-class="top-tooltip"
+      transition="fade-transition"
+      nudge-right="3"
+    >
+      <template #activator="{ on, attrs }">
+        <span
+          v-bind="attrs"
+          class="tool-tip-text"
+          v-on="on"
+        >special resolution</span>
+      </template>
+      <span class="tool-tip-content">
+        Special Resolution -  A decision voted on by the voting members of a Cooperative Association.
+      </span>
+    </v-tooltip>
+    for this amendment
```

Comment:
Do you have a design of what this should look like?

Should there be a period at the end of line 21?

Reply:
Fixed, thanks.